Gemfile

```diff
@@ -21,3 +21,5 @@ gem 'scraperwiki', github: 'openaustralia/scraperwiki-ruby',
 gem 'table_unspanner', github: 'everypolitician/table_unspanner'
 gem 'vcr'
 gem 'webmock'
+gem 'wikidata-fetcher', '>= 0.19.1', github: 'everypolitician/wikidata-fetcher'
```

Comment:
From what I can tell we're only using the `wikidata-client` dependency from `wikidata-fetcher`, so perhaps it would be better to have a dependency on that instead?